.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,77 @@
+name: Build and test wheels
+
+on: [push, pull_request]
+
+jobs:
+  wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macOS-10.15]
+        arch: [auto]
+        include:
+          - os: windows-2016
+            cibw-arch: AMD64
+            cmake-generator: "Visual Studio 15 2017 Win64"
+          - os: windows-2016
+            cibw-arch: x86
+            cmake-generator: "Visual Studio 15 2017"
+          - os: ubuntu-20.04
+            arch: aarch64
+
+    steps:
+      - name: Disable autocrlf
+        run: |
+            git config --global core.autocrlf false
+
+      - uses: actions/checkout@v2
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v2
+
+      - uses: docker/setup-qemu-action@v1
+        if: ${{ matrix.arch == 'aarch64' }}
+        name: Set up QEMU
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel
+
+      - name: Build wheels
```

Comment:
To consider: these jobs are really long, I don't know why though (tests setup takes over 5 minutes??)
40 minutes for ubuntu one to finish. 1h 20m for the qemu one. Or even longer.

 Github Actions are not degraded apparently, but something is weird. Isn't that waay too long?

Should we create a faster python jobs and build wheels only on merge or do we accept that as dlisio development is kinda stopped?

Reply:
They are, but as there is no active development no it's fine.